pysph/examples/dam_break_2d.py

```diff
@@ -146,7 +195,23 @@ def create_particles(self):
         if self.options.scheme == 'iisph':
             # the default position tends to cause the particles to be pushed
             # away from the wall, so displacing it by a tiny amount helps.
-            fluid.x += self.dx/4
+            fluid.x += self.dx / 4
+
+        # Adding extra properties for kernel correction
```

Comment:
Can these be added only if the correction is required instead of always?

Reply:
Added if conditions for each type of kernel correction